src/App.tsx

```diff
@@ -15,47 +16,153 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  // To avoid the original array mutation
```

Comment:
It is not important, but just make sure you remove the comments before git push the code.
It good practice comment your code, well done!,  but just to mention, in production, the code should not have comments. I know this is not production, but just leave it without them.

Reply:
@kenvastolord it is really important, since those comments in the real project can really confuse